README.md

```diff
@@ -256,22 +267,91 @@ kpt alpha repo register \
 It is also possible to set a different branch and directory for packages within
 the repository; see `kpt alpha repo register --help` for more.
 
+## Creating a Workload Cluster
+
+Workload clusters are  those clusters that do not contain the Nephio system itself,
+but instead are intended to run the workloads deployed via Nephio.
+
+```
+# Create a workload cluster
+gcloud container clusters create-auto --region us-central1 nephio-edge-01
+# This will take a few minutes
+# Once it returns, configure kubectl with the credentials for the cluster
+gcloud container clusters get-credentials --region us-central1 nephio-edge-01
+
+# See the nodes in your new cluster
+kubectl get nodes
+NAME                                            STATUS   ROLES    AGE    VERSION
+gk3-nephio-edge-01-default-pool-0e8b5852-c1w5   Ready    <none>   1m5s   v1.22.10-gke.600
+gk3-nephio-edge-01-default-pool-792b3c85-p0f4   Ready    <none>   1m5s   v1.22.10-gke.600
+```
+
 ## Installing Config Sync in Workload Clusters
 
-Workload clusters - those clusters that do not contain the Nephio system itself,
-but instead are intended to run the workloads deployed via Nephio - must run
-Config Sync to get their workloads from their deployment repositories.
+Workload clusters must run Config Sync to get their workloads from their deployment repositories.
 
 The package
-[nephio-configsync](https://github.com/nephio-project/nephio-packages.git/nephio-configsync)
+[nephio-configsync](https://github.com/nephio-project/nephio-packages/tree/main/nephio-configsync)
 is intended for installing Config Sync in those clusters. To use this package,
 you will need to update the RootSync resource with the repository and
 authentication needed for your environment.
 
 Example:
 ```
-kpt pkg get --for-deployment https://github.com/nephio-project/nephio-packages.git/nephio-configsync test-deploy-edge-01
-kpt fn render test-deploy-edge-01
-kpt live init test-deploy-edge-01
-kpt live --context=gke_jbelamaric-dev_us-central1_nephio-edge-01 apply test-deploy-edge-01 --reconcile-timeout=15m --output=table
+kpt pkg get --for-deployment https://github.com/nephio-project/nephio-packages.git/nephio-configsync nephio-test-deploy-01
```

Comment:
Nice! May want to mention that you can use a function to do this - or you can simply edit the file with your text editor. 

The function method is amenable to automation - you could inject the GITHUB_USERNAME env var for example and automate what you have here.

On the other hand, you can do it manually. You can pull the package, edit it with your GitHub username, and then push it to your private catalog. If you `kpt pkg get` from there, then there will be no more editing needed for subsequent workload clusters.

One of the things we're trying to do with CaD is break the mindset of "you must only use code to edit the config". In fact, there is no reason you can't just change the resources as you wish, directly.

What isn't clear is 1) how to guide the user in this sort of editing; 2) when it's best to use a function for this or just leave it to the user. For 2), you have used a general purpose "find and replace" function. That works. But one question is whether there should be a special-purpose function for managing the RootSync resource. This might facilitate easier configuration of that object. Or...not. Maybe it's just easier to edit it directly? It's not clear to me.

Reply:
When there are multiple resources involved that must remain consistent in some way (think, Service & Deployment labels and selectors), then a function can clearly improve things. It's not clear when there is a single resource, because then the function inputs themselves are just a different API but have to capture all the same information. So, why not just use the main API (the K8s resource itself) directly?

Reply:
Great points. I have updated the description to highlight that manual edits are perfectly acceptable in the CaD world.
Interesting question of 'to manual edit or not' in this case. I reckon the fact that it is such a simple change would favour manual. However, since the pkg would have to be published (and adjusted) to each workload cluster I would probably cast my vote on automation.